web/src/components/ResourcesDialog.tsx

```diff
@@ -152,10 +152,10 @@ const ResourcesDialog: React.FC<Props> = (props: Props) => {
                   <span className="field-text">{resource.type}</span>
                   <div className="buttons-container">
                     <Dropdown className="actions-dropdown">
-                      <button onClick={() => handlPreviewBtnClick(resource)}>Preview</button>
-                      <button onClick={() => handleCopyResourceLinkBtnClick(resource)}>Copy Link</button>
+                      <button onClick={() => handlPreviewBtnClick(resource)}>{t("resources.preview")}</button>
+                      <button onClick={() => handleCopyResourceLinkBtnClick(resource)}>{t("resources.copy-link")}</button>
                       <button className="delete-btn" onClick={() => handleDeleteResourceBtnClick(resource)}>
-                        Delete
+                        {t("resources.delete")}
```

Comment:
I think we can use the exist words in common: https://github.com/usememos/memos/blob/main/web/src/locales/en.json#L18
```suggestion
                        {t("common.delete")}
```

Reply:
Agreed.